src/main/features/claim/guards/claimEligibilityGuard.ts

```diff
@@ -0,0 +1,47 @@
+import * as express from 'express'
+
+import { GuardFactory } from 'response/guards/guardFactory'
+
+import { Draft } from '@hmcts/draft-store-client'
+import { DraftClaim } from 'drafts/models/draftClaim'
+import { DraftService } from 'services/draftService'
+
+import { CookieEligibilityStore } from 'eligibility/store'
+
+import { Paths as EligibilityPaths } from 'eligibility/paths'
+
+const eligibilityStore = new CookieEligibilityStore()
+
+export class ClaimEligibilityGuard {
+  /**
```

Comment:
:(, probably need a quick way to skip eligibility then, because its slow and painful to fill out

Reply:
We could disable it locally by feature flag or relay on quicker mechanism of populating draft ready to use.

I also thought that we could need to shut down the service for new users by redirecting everyone to MCOL in case we had serious performance issues with overwhelming pubic beta volumes. Should not be difficult to do it in form of emergency release.

Reply:
emergency release is an old term,
you mean make a code change?

We need to do something about it next sprint